oragono.yaml

```diff
@@ -198,6 +198,15 @@ accounts:
     # PASS as well, so it can be configured to authenticate with SASL only.
     skip-server-password: false
 
+    require-sasl:
```

Comment:
can we please switch up this whole config block for something like this? just fits in a liiittle bit more cleanly with the existing style:
```yaml
    # require-sasl controls whether clients are required to have accounts
    # (and sign into them using SASL) to connect to the server
    require-sasl:
        # if this is enabled, all clients must authenticate with SASL while connecting
        enabled: false
        
        # IPs/CIDRs which are exempted from the account requirement
        exempted:
            - "localhost"
            # - '127.0.0.2'
            # - '10.10.0.0/16'
```